### PR TITLE
miner powercreep starring 150 damage pointblank shotgun

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
replaces the blood contract with a super shotgun on bubblegum's loot table
featuring codersprites

## Why It's Good For The Game
gets rid of blood contract but still manages to powercreep mining pvp on station, aw yeah

## Changelog
:cl:
add: a super combat shotgun that loads and fires 2 shells at a time
balance: replaces bubblegum's blood contract drop with the super shotgun
/:cl:

### CODE REVIEW

**Comment by YakumoChen** [comment]
image

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -1028,6 +1028,17 @@
 	log_combat(user, L, "took out a blood contract on", src)
 	qdel(src)
 
+/obj/item/gun/ballistic/revolver/doublebarrel/super
```
Comment (Trilbyspaceclone) [inline code/modules/mining/lavaland/necropolis_chests.dm:1031]: can you make this fit onto the H.E.C.K suit?

Comment (ghost) [reply code/modules/mining/lavaland/necropolis_chests.dm:1031]: none of the good lavaland armor holds all gun subtypes so if they want to not be holding it at all times they have to trade for weaker sec armor is the idea

**Review by silicons — APPROVED** [review]

**Comment by ghost** [comment]
god bless doomerica

**Pull request merged**